### DISCUSSION TITLE
What breaking changes were introduced in 2.0?

### DISCUSSION THREAD

**Discussion by daxelrod-datto** [kubescape/kubescape#502] [Q&A] [answered]
Hi! I'm currently using the Kubescape CLI tool at version 1.0.139 and planning an upgrade to a more recent version.

Did the major version bump from 1 to 2 imply that there were breaking changes? If so, what were these breaking changes and what do I need to do to adapt my usage of `kubescape download framework` and `kubescape scan framework` against helm charts to the new version?

The [release notes](https://github.com/armosec/kubescape/releases/tag/v2.0.140) just mention "New kubescape major version" and link to #330 , which contains a large number of changes, and I'm having trouble figuring which ones (if any) are breaking changes.

It's possible I'm missing documentation about this somewhere else, but I haven't been able to find it yet.

Thank you for all the work you do to make Kubescape awesome!

**Comment by dwertent** [comment, marked as answer]
Hi @daxelrod-datto, the major version is because we made some structure changes in the kubescape repo, and we added the ability of running kubescape as a microservice so we can support [Prometheus integration](https://hub.armo.cloud/docs/installation-of-armo-in-cluster#prometheus-support).
**The `download` and `scan` commands where not broken.**

FYI You can run `kubescape download artifacts -o <path>` for downloading all relevant artifacts, and scan `kubescape scan --use-artifacts-from <path>` so there is no need of specifying a framework name

**Reply by daxelrod-datto** [reply]
Awesome, thank you!
